recipes-soletta/soletta/soletta_git.bb

```diff
@@ -6,6 +6,7 @@ DESCRIPTION = "Soletta library and modules"
 SECTION = "examples"
 DEPENDS = "glib-2.0 libpcre pkgconfig python3-jsonschema-native icu curl libmicrohttpd mosquitto nodejs"
 DEPENDS += " ${@bb.utils.contains('DISTRO_FEATURES','systemd','systemd','',d)}"
+DEPENDS += " ${@bb.utils.contains('DISTRO_FEATURES','x11','gtk+3','',d)}"
```

Comment:
At least that's the theoretical purpose of PACKAGECONFIG: declaration of conditional dependencies plus providing configure options.

But personally I saw situations when even putting something to DEPENDS doesn't guaranty the thing is built by the time the recipe starts do_configure.